packages/protocol/engine/tangle/booker/booker.go

```diff
@@ -145,6 +145,31 @@ func (b *Booker) PayloadConflictIDs(block *virtualvoting.Block) (conflictIDs utx
 	return
 }
 
+// PayloadConflictID returns the ConflictID of the conflicting payload contained in the given Block without conflicts from the UTXO past cone.
+func (b *Booker) PayloadConflictID(block *virtualvoting.Block) (conflictID utxo.TransactionID, conflictingConflictIDs utxo.TransactionIDs, isTransaction bool) {
+	if b.BlockDAG.EvictionState.InEvictedEpoch(block.ID()) {
+		return conflictID, conflictingConflictIDs, false
+	}
+
+	transaction, isTransaction := block.Transaction()
+	if !isTransaction {
+		return conflictID, conflictingConflictIDs, false
```

Comment:
`conflictingConflictIDs` is nil here. I'd prefer to return a new empty set or explicitly nil